logging/src/commonTest/kotlin/io/spine/logging/LogSiteInjectorSpec.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023, The Flogger Authors; 2025, TeamDev. All rights reserved.
+ * Copyright 2025, TeamDev. All rights reserved.
```

Comment:
I think the Flogger authors should be credited here.

Reply:
It's vice versa. We need to adjust the statement in `LogSiteInjector` class not to mention the Flogger Authors, because it was introduced in our code instead of pseudo-deprecation from the original code.